tests/integration/interface/fortran/version_fortran_tests.F

```diff
@@ -0,0 +1,52 @@
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+! Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+! project contributors. See the COPYRIGHT file for details.
+!
+! SPDX-License-Identifier: (MIT)
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+module umpire_fortran_version_tests
+
+  use iso_c_binding
+  use fruit
+  use umpire_mod
+  implicit none
+
+  contains
+
+      subroutine test_version
+          use iso_c_binding
+
+          integer(C_SIZE_T) :: major_ver, minor_ver, patch_ver
+
+          major_ver = get_major_version()
+          minor_ver = get_minor_version()
+          patch_ver = get_patch_version()
+
+          call assert_true(major_ver .eq. 5)
+          call assert_true(minor_ver .eq. 0)
+          call assert_true(patch_ver .eq. 0)
```

Comment:
This is hard coded for version 5.0.0 - so with each new major release, this will be changed? Should that be added to our release notes/checklist on confluence?

Reply:
Yes, that's a good idea. The reason is that I didn't want to figure out how to use the version macros in Fortran :D since they are a C preprocessor thing.

Reply:
makes sense lol

Reply:
Isn't fortran code also run through a preprocessor that is basically the same as the c preprocessor?

Reply:
Maybe you could make this a .F.in?